kpi/tests/api/v2/test_api_collections.py

```diff
@@ -307,6 +307,48 @@ def test_collection_subscribe(self):
             response.data['results'][0]['url'].endswith(pub_coll_url)
         )
 
+    def test_get_subscribed_collection(self):
+        public_collection = Asset.objects.create(
+            asset_type=ASSET_TYPE_COLLECTION,
+            name='public collection',
+            owner=self.someuser,
+        )
+        public_collection_asset = Asset.objects.create(
+            asset_type=ASSET_TYPE_TEMPLATE,
+            name='public asset',
+            owner=self.someuser,
+            parent_id=public_collection.pk
+        )
+        public_collection.assign_perm(AnonymousUser(), PERM_DISCOVER_ASSET)
+
+        self.login_as_other_user(username="anotheruser", password="anotheruser")
+
+        asset_list_url = reverse(self._get_endpoint('asset-list'))
+        coll_list_url = f'{asset_list_url}?q=asset_type:collection'
+        sub_list_url = reverse(self._get_endpoint('userassetsubscription-list'))
+        sub_coll_url = f'{asset_list_url}?q=parent__uid:{public_collection.uid}'
```

Comment:
nitpick: suggest more verbose names since `sub` in `sub_list_url` means `subscriptions` but `subscribed` in `sub_coll_url`